src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/appsettings.Docker.json

```diff
@@ -3,8 +3,8 @@
     "DefaultConnection": "Server=db;Database=aspnet-DevAdventCalendarCompetition-33CBD544-AB08-45ED-A959-D039E993B165;User=SA;Password=DevCal123!."
   },
   "StringHasher": {
-    "Iterations": "",
-    "Salt": ""
+    "Iterations": "100",
```

Comment:
ok, sure. I can't run the app without it, so I thought that I'll help others by this.

that's the big drawback of this repo for me - I can't test my changes, because it has a lot of configuration that I don't know, that I can't supply :(

Reply:
Ok, I understand. Could you create a new feature issue with a description of which parts of the project configuration were missing or needed to create manually?  We will think about a better solution for local environment configuration.